### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by Enekwestanley**
This is project to add the contact section of the Portfolio Page.

Technologies used;

- Github
- Vs code

Built with;

- HTML
- CSS

### CODE REVIEW

**Review by rotimiazeez — COMMENTED** [review]
# Hi Team, 

### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that That's not passing the linter checks. Kindly fix this and submit it again.💪🏽

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -200,6 +200,27 @@ <h2 class="big-text">About Myself</h2>
...
         </div>
     </section>
+    <section class="contact-section">
+      <div class="contact-Me">
+        <h3 class="contact">Contact Me</h3>
+        <p class="last-text">If you have an application you are interested in developing, a feature that you need built or a project that
+          needs coding. I’d love to help with it</p>
+        <form action="https://formspree.io/f/xvolqjkn" method="post" id="contact-form">
+          <label for="name" id="name-label"></label>
+          <input  id="name" type="text" required placeholder="Enter your Name">
```
Comment (medaminedev66) [inline index.html:210]: - [ ] As stated in the project requirements: Please ensure that the name input accepts up to 30 chars. You can do this by adding the `maxlength="30"` attribute.

File: index.html [hunk 2]
```diff
@@ -200,6 +200,27 @@ <h2 class="big-text">About Myself</h2>
...
+          <label for="email" id="email-label"></label>
+          <input id="email" type="email" required placeholder="example@gmail">
+          <label for="message"></label><br>
+          <textarea name="message" id="message" cols="30" rows="10" required placeholder="Write your messge here"></textarea>
```
Comment (medaminedev66) [inline index.html:214]: - [ ] Message input also should have `maxlength="500"` attribute.

File: index.html [hunk 3]
```diff
@@ -200,6 +200,27 @@ <h2 class="big-text">About Myself</h2>
...
+          <button type="submit" class="touch">Get in Touch</button>
```
Comment (medaminedev66) [inline index.html:215]: - [ ] Your form button design is great 👍. Can you please also follow the button styling: Active, Hover, and Pressed (see Figma design below). This makes your User Experience better.
image

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @aimalamiri and @Enekwestanley 😃, :wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- ✔️The repo looks professional
- ✔️Git/GitHub flow best practices have been followed
- ✔️There are no linter errors


## Required Changes ♻️

_See the inline comments below for more required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @Enekwestanley ** , 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Your Design looks awesome, keep rocking :rocket: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**